### DISCUSSION TITLE
Unable to connect to transmitter - please help!

### DISCUSSION THREAD

**Discussion by cobwebT1D** [NightscoutFoundation/xDrip#3328] [Bug reports]
Version - 23ccbd1-2023.07.16
Data Source - Dex G6 Native
Phone - Pixel 7a

This morning xdrip stopped receiving data from my sensor, which was about half way through it's life. After doing the usual - changing the transmitter ID, turning off bluetooth, turning off the phone, I inserted a new sensor. But the phone still can't find the transmitter. In hindsight it was obvious it was a transmitter issue not a sensor issue - but I only have one Dexcom left!

I'm running Android AAPS so would really like to fix this asap.

Thank you!

**Comment by Navid200** [comment]
Please post a screenshot of the status page and you can blackout the transmitter ID if you like.  This is the status page I mean: https://navid200.github.io/xDrip/docs/StatusG5G6.html
